### ISSUE TITLE
Zoom/Skype/Teamviewer/Slack/Teams (video conferencing/screen sharing software)

### ISSUE THREAD

**Issue by alexanderadam** [junaid33/opensource.builders#56]
# Jitsi video conferencing solution

It's awesome and based of great technologies such as XMPP and WebRTC but I'm not quite sure which "alternative" it could be, though. :wink: 

https://jitsi.org/

Source codes are [here](https://github.com/jitsi?type=source).

**Comment by jcklpe** [comment]
@alexanderadam Hi, sorry, the closed source project should be in the title of the ticket, and the comments should provide open source alternatives. You have it backwards. Please edit to follow the form correctly.

**Comment by alexanderadam** [comment]
I just saw that [junaid33 added Jitsi 11 days ago](https://github.com/junaid33/opensource.builders/commit/b603a152334a5fe4c64ab96c35e7b1df417048e9). I guess he just hadn't a look at this issue since February.

**Comment by jcklpe** [comment]
Yeah I'm keeping it open so people don't dupe the other similar features. Not sure what we'll do long term about that, but for now keeping them as placeholders.

**Comment by jcklpe** [comment]
https://github.com/luimi/openMeet

**Comment by jcklpe** [comment]
bigbluebutton and NextCloud Talk.

**Comment by luketainton** [comment]
Spreed WebRTC

**Comment by jcklpe** [comment]
https://github.com/hashicorp/boundary  
  
Not so much screen share as it is direct remote access like teamviewer

**Comment by pierregeekme** [comment]
The alternative Bigbluetton :
website : https://bigbluebutton.org/
repository github : https://github.com/bigbluebutton/bigbluebutton

Integration with Mattermost it's possible and easy.

Thank you

**Comment by luketainton** [comment]
DWService - https://www.dwservice.net
